requirements/base.txt

```diff
@@ -1,48 +1,48 @@
 bs4
 ccxt
 cryptocompare==0.6
-cryptography==2.1.4
+cryptography==2.2.2
 django==2.0.4
-django-bootstrap3==8.2.3
+django-bootstrap3==9.1.0
```

Comment:
@mbeacom Thoughts on shifting to [django-bootstrap4](https://github.com/zostera/django-bootstrap4/) ?

Reply:
I'd love to... but I didn't want to make any **major** changes to the python packages in this PR.  The main thing here is the upgrade from `jessie` to `stretch` for LTS and upgraded system packages like `OpenSSL` jumping to `1.1`

We can definitely raise a PR following this to upgrade to `django-bootstrap4` - Thanks for looking out @thelostone-mc 😛 - If no one has objections, we could try it in this PR.  Looking to merge this following #668

Reply:
Let's give it a shot post #668 merge :D

Reply:
We can probably just rip out `django-bootstrap3` since it's unused throughout the codebase.